esphome/components/slow_pwm/slow_pwm_output.cpp

```diff
@@ -0,0 +1,40 @@
+#include "slow_pwm_output.h"
+#include "esphome/core/log.h"
+
+namespace esphome {
+namespace slow_pwm {
+
+static const char *TAG = "output.slow_pwm";
+
+void SlowPWMOutput::setup() {
+  this->pin_->setup();
+  this->turn_off();
+}
+
+void SlowPWMOutput::loop() {
+  unsigned long now = millis();
+  float scaled_state = this->state_ * this->period_;
+
+  if (now - this->period_start_time_ > this->period_) {
+    ESP_LOGD(TAG, "End of period. State: %f, Scaled state: %f", this->state_, scaled_state);
```

Comment:
```suggestion
    ESP_LOGVV(TAG, "End of period. State: %f, Scaled state: %f", this->state_, scaled_state);
```

Debug is certainly too high of a verbosity here (debug is default level, not a great decision in hindsight but it is that way now).

This message will appear once every period, and spam the logs quite a bit.